.travis.yml

```diff
@@ -2,9 +2,7 @@ language: java
 
 jdk:
   - openjdk8
-  - openjdk9
-  - openjdk11
-  - openjdk12
```

Comment:
Why not leave 9 and 11?

Reply:
Because they are no longer supported by any of the vendors.

Reply:
I mean, just Java 9 and 12. I’ll restore 11